src/Combinatorics/Graphs/functions.jl

```diff
@@ -507,6 +507,27 @@ function neighbors(g::Graph{T}, v::Int64) where {T <: Union{Directed, Undirected
     return [x+1 for x in result]
 end
 
+@doc raw"""
+    degree(g::Graph{T}; v::Int64) where {T <: Union{Directed, Undirected}}
```

Comment:
Total nitpick: With the semicolon I would expect `v` to be a keyword argument. In other places, we use
```suggestion
    degree(g::Graph{T}[, v::Int64]) where {T <: Union{Directed, Undirected}}
```
for "optional" arguments.

Reply:
Thanks, I adapted accordingly.